packages/amplication-client/src/util/analytics-events.types.ts

```diff
@@ -0,0 +1,94 @@
+  WorkspaceMemberInvitationRevoke = "revokeInvitation",
+  WorkspaceMemberFromWorkspaceDelete = "deleteUserFromWorkspace",
+
+  // menu
+  SupportButtonClick = "supportButtonClick",
+  SupportDocsClick = "supportDocsClick",
+  SupportCommunityClick = "supportCommunityClick",
+  SupportFeatureRequestClick = "supportFeatureRequestClick",
+  SupportIssueClick = "supportIssueClick",
+  ThemeSet = "setTheme",
+
+  // tiles that exists in all kinds of resources (service, message broker, etc.)
+  DocsTileClick = "docsTileClick",
+  FeatureRequestTileClick = "featureRequestTileClick",
+  NewVersionTileFixEntitiesClick = "newVersionTileClick-fixEntities",
+  RolesTileClick = "rolesTileClick",
+
+  // project & project configuration
+  ProjectConfigurationsSettingsUpdate = "updateProjectConfigurationsSettings",
+  ProjectCreate = "createProject",
+
+  // service
+  ServiceSettingsUpdate = "updateServiceSettings",
+
+  // resource
+  ResourceFromFileCreate = "createResourceFromFile",
+  ResourceFromSampleCreate = "createResourceFromSample",
+  ResourceFromScratchCreate = "createResourceFromScratch",
+  ResourceInfoUpdate = "updateResourceInfo",
+  ResourceBuild = "buildResource",
+  ResourceDelete = "deleteResource",
+  ResourceCardClick = "resourceCardClick",
+
+  // entity
+  EntityCreate = "createEntity",
+  EntityUpdate = "updateEntity",
+  EntityFieldCreate = "createEntityField",
+  EntityFieldUpdate = "updateEntityField",
+  RelatedEntityFix = "fixRelatedEntity",
+  EntitiesTileClick = "entitiesTileClick",
+
+  // message broker
+  MessageBrokerConnectedServicesTileClick = "messageBrokerConnectedServicesTileClick",
+  MessageBrokerTopicsTileClick = "messageBrokerTopicsTileClick",
+  MessageBrokerErrorCreate = "messageBrokerErrorCreate",
+  MessageBrokerCreateClick = "createMessageBrokerClick",
+  BackToProjectsClick = "backToProjectsClick",
+
+  // topic
+  TopicCreate = "topicCreate",
+  TopicCreateFailed = "topicCreateFailed",
+  TopicNameEdit = "topicNameEdit",
+  TopicDisplayNameEdit = "topicDisplayNameEdit",
+  TopicDescriptionEdit = "topicDescriptionEdit",
+  TopicsSearch = "topicsSearch",
+
+  // commit & build & pending changes
+  CommitCreate = "commit",
+  CommitListBuildIdClick = "commitListBuildIdClick",
+  LastCommitIdClick = "lastCommitIdClick",
+  PendingChangesDiscard = "discardPendingChanges",
+  LastBuildIdClick = "lastBuildIdClick",
+
+  // code view
+  // TODO: ask about these names
+  CodeViewTileClick = "viewCodeViewTileClick",
+  GithubCodeViewClick = "openGithubCodeView",
+
+  // GitHub
+  GitHubAuthResourceStart = "startAuthResourceWithGitHub",
+  GitHubAuthResourceComplete = "completeAuthResourceWithGitHub",
+  GitHubRepositoryCreate = "createGitRepository",
+  GithubRepositoryChange = "changeGithubRepository",
+  GithubOpenPullRequest = "openGithubPullRequest",
+  // TODO: ask about these names
+  GitHubRepositorySync = "selectGitRepository",
+  GithubRepoSync = "selectGithubRepo",
```

Comment:
I think these two events are the same, but somehow they got different names

Reply:
@yuval-hazaz  @GreenMachine01 I think you guys will know more here.

Reply:
@abrl91 it looks like you are correct, but please check where these events are being used